react/components/admin/store/index.tsx

```diff
@@ -48,6 +56,16 @@ const Store: React.FC<InjectedIntlProps> = ({ intl }) => {
           active={activeTab === 'advanced'}
           onClick={handleAdvancedOpen}
           label={intl.formatMessage(messages.advanced)}
+        >
+          <div className="pv6">
+            <StoreForm advanced />
```

Comment:
Where does `StoreForm` come from? Would it be difficult creating a different component for the `advanced` page?

If so, could you use an enum instead of a boolean (i.e. `<StoreForm page="advanced" />`)

Reply:
Good idea!